config.go

```diff
@@ -369,6 +378,15 @@ func SkipBloomFilters(skip bool) FileOption {
 	return fileOption(func(config *FileConfig) { config.SkipBloomFilters = skip })
 }
 
+// SetPageReadMode is a file configuration option which controls the way pages
+// are read. Currently the only two options are PageReadModeAsync and PageReadModeSync
+// which control whether or not pages are loaded asynchronously.
+//
+// Defaults to PageReadModeAsync.
+func SetPageReadMode(mode PageReadMode) FileOption {
```

Comment:
To stay consistent with other options, maybe rename?
```suggestion
func FileReadMode(mode ReadMode) FileOption {
```

Reply:
done